pkg/kubectl/cmd/taint.go

```diff
@@ -102,7 +91,8 @@ func NewCmdTaint(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 				cmdutil.CheckErr(err)
 			}
 		},
-		ValidArgs: validArgs,
+		ValidArgs:  []string{"node"},
+		ArgAliases: []string{"nodes"},
```

Comment:
for this command, we do not support "no" at present. I guess this `kubectl.ResourceAliases(validArgs)` will introduce three names: "no", "node" and "nodes"